From 9b9ea2402d1db385a7f240aa194ea05fceb29e81 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Fri, 2 Oct 2026 00:37:26 +0800
Subject: [PATCH 1/2] AOSCOS: mainwindow: hide playlist by default

We tend to be using local players in much simpler fashion nowadays and
having a playlist showing constantly seems non-productive.

Hide it by default.

Signed-off-by: Mingcong Bai <jeffbai@aosc.io>
---
 src/mainwindow.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/mainwindow.cpp b/src/mainwindow.cpp
index fe734398..60281a34 100644
--- a/src/mainwindow.cpp
+++ b/src/mainwindow.cpp
@@ -216,7 +216,7 @@ void MainWindow::setState(const QVariantMap &map)
     UNWRAP(ui->actionViewHideStatistics, false);
     UNWRAP(ui->actionViewHideStatus, true);
     UNWRAP(ui->actionViewHideSubresync, false);
-    UNWRAP(ui->actionViewHidePlaylist, true);
+    UNWRAP(ui->actionViewHidePlaylist, false);
     UNWRAP(ui->actionViewHideCapture, false);
     UNWRAP(ui->actionViewHideNavigation, false);
     UNWRAP(ui->actionViewHideLog, false);
-- 
2.55.0

